packages/components/src/CreditCard/Update.purs

```diff
@@ -48,55 +48,53 @@ initialState :: State
 initialState =
   { poller: pure unit
   , updateState: ChooseCreditCard
-  , chosenCreditCard: Nothing
   }
 
 component :: React.Component Props
-component = React.createComponent "update"
+component = React.createComponent "Update"
```

Comment:
It's an internal identifier, does capitalizing it affect anything?

Reply:
Not really, but I noticed that it was lower case while all other components have upper case :laughing: